gitbutler-app/src/projects/controller.rs

```diff
@@ -201,6 +201,10 @@ impl Controller {
             tracing::error!(project_id = %id, ?error, "failed to remove project data",);
         }
 
+        if let Err(error) = std::fs::remove_file(project.path.join(".git/gitbutler.json")) {
+            tracing::error!(project_id = %project.id, ?error, "failed to remove .git/gitbutler.json data",);
+        }
+
```

Comment:
I got bit confused with [this function](https://github.com/gitbutlerapp/gitbutler/blob/master/gitbutler-app/src/gb_repository/repository.rs#L542) so not sure whether proposed fix is good enough or not

Reply:
@krlvi this seems reasonable to me, what do you think?